package.json

```diff
@@ -71,11 +73,13 @@
   },
   "devDependencies": {
     "@types/jest": "^23.3.1",
+    "@types/material-ui": "^0.21.5",
```

Comment:
We don't use it

Reply:
Deleted material-ui with all dependencies